client/src/pages/QuizLoginPage.tsx

```diff
@@ -0,0 +1,55 @@
+import { useParams } from "react-router-dom";
+import H1 from "../components/typography/H1";
+import { useEffect, useState } from "react";
+import LoginForm from "../components/quiz/LoginForm";
+// import { default as quizData } from "../data/QuizMockData.json";
+
+function QuizLoginPage() {
+  // Id wordt gebruikt om bij de juiste quiz te komen, maar is nu nog niet geïmplementeerd.
```

Comment:
Aha hier staat uitgelegd hoe en wat.

Logica mag eigenlijk niet in de Page, maar moet in de component zelf.

Eigenlijk wil je dat logica alleen in components wordt toegepast en dan een page alleen dingen importeert of props meegeeft vanuit een mogelijke store etc.

Reply:
Stel anders een folderstructuur aan waar alle met z'n alle aan kunnen houden. Welke folderstructuur houd je hiervoor aan in je projecten? Heb je een artikel die ik kan lezen? 
En Hoe bepaal je wat hoort in een page en in een component?